roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ds-ingress-nginx-controller.yml.j2

```diff
@@ -95,6 +95,27 @@ spec:
               containerPort: 10254
 {% if not ingress_nginx_host_network %}
               hostPort: {{ ingress_nginx_metrics_port }}
+{% endif %}
+{% if ingress_nginx_cpu_limit != "" or ingress_nginx_memory_limit !="" or ingress_nginx_cpu_requests !="" or ingress_nginx_memory_requests !="" %}
```

Comment:
Personally I would prefer something more like a dict with the appropriate keys which can be converted using `to_nice_yaml` (see https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#formatting-data-yaml-and-json)

Reply:
Nevermind, I just saw the comment from @oomichi on consistency with other components. I still don't quite like our approach to this but it makes sens to follow an already established practice.

Also please squash your commits into a single one since it does affect a single change.

Reply:
In the previous version, it was so, but @oomichi recommended making various variables for the consistency of Kubespray